contracts/scripts/config/update-round.config.ts

```diff
@@ -0,0 +1,31 @@
+import metadataJson from './assets/grant-round-metadata.json';
+
+type NetworkParams = {
+  roundAddress: string;
+  ipfsRetrievalEndpoint: string;
+  roundLogoPath: string; // this path must be relative to the 'contracts' directory
+  metadataJson: Record<string, any>;
+};
+
+type DeployParams = Record<string, NetworkParams>;
+
+const params: DeployParams = {
+  localhost: {
+    // expected address using app.ts w/ hardhat default account
+    //roundAddress: '0x8b4091997e3ebb87be90ced3e50d8bb27e1dc742',
+
+    // expected address using deploy-poc.ts w/ hardhat default account
+    roundAddress: '0x851248dBF29D723f3A12EA9739A9D1c88d1c7faa',
+    ipfsRetrievalEndpoint: 'https://ipfs-dev.fleek.co/ipfs',
+    roundLogoPath: './scripts/config/assets/dgrants-placeholder.png',
+    metadataJson,
+  },
+  mainnet: {
+    roundAddress: '0x0000000000000000000000000000000000000000',
+    ipfsRetrievalEndpoint: 'https://ipfs-dev.fleek.co/ipfs',
+    roundLogoPath: './scripts/dgrants-placeholder.png',
```

Comment:
I think we need to change these paths now since this relative path is no longer accurate?

Reply:
```suggestion
    roundLogoPath: '../dgrants-placeholder.png',
```

Reply:
Actually no, we don't do that. At the time this path is used, it's in the context of the hardhat execution environment. See this comment: https://github.com/dcgtc/dgrants/pull/228/files#diff-4e111680456eadf826b03b37ee14a4b29cbae04d7ba601780d839a8a3f80ffd0R6

Reply:
Oh, I see, @mds1 you're right that I missed the mainnet paths, but @thelostone-mc your suggested update is wrong for the reason I explained. Will push a fix.